parameterOptimization/parameterOptimization.md

```diff
@@ -24,6 +24,9 @@ https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server
 
 ### Main suggestions:
 
+SET work_mem = '32MB' prevents memory from spilling to the disk.
```

Comment:
Could you please include a description of what was causing the spilling onto disk and what was measured in the times given.